librz/util/sys.c

```diff
@@ -1378,6 +1378,9 @@ RZ_API void rz_sys_info_free(RSysInfo *si) {
 
 #if __UNIX__ && HAVE_PIPE2
 #include <fcntl.h>
+#ifndef _GNU_SOURCE
```

Comment:
this is not necessary. It is already defined in meson.build

Reply:
It was necessary for the ACR-based `sys/debian.sh`. But because we removed the use of it, I will remove those lines too.